reference/managed-delivery/getting-started.md

```diff
@@ -7,15 +7,15 @@ sidebar:
 
 {% include toc %}
 
-You'll be interacting with Spinnaker's Managed Delivery using curl and the Spinnaker api for now.
+You'll be interacting with Spinnaker's Managed Delivery using `curl` and the Spinnaker API for now.
```

Comment:
Since you mention the UI below, should we expand this a bit to say something like “UI support for Managed Delivery is limited at this point, so for certain use cases you’ll be interacting with Spinnaker using curl...”?

Reply:
yep good call thanks, I'm going to mention both the UI and 'version control' since technically these resource CRUD operations will probably always be done via the command line or version control integration.